atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SerializableTransaction.java

```diff
@@ -304,15 +305,7 @@ private void updateRangeEnd(Cell end) {
     @Override
     @Idempotent
     public Map<Cell, byte[]> get(TableReference tableRef, Set<Cell> cells) {
-        try {
-            return getWithLoader(
-                            tableRef,
-                            cells,
-                            (tableReference, toRead) -> Futures.immediateFuture(super.get(tableRef, toRead)))
-                    .get();
-        } catch (InterruptedException | ExecutionException e) {
-            throw Throwables.rewrapAndThrowUncheckedException(e.getCause());
-        }
+        return super.get(tableRef, cells);
```

Comment:
Why aren't we getting with loader? We'll surely still need to do R-W conflicts for normal remote reads?